### PR TITLE
get-packages

### PR DESCRIPTION
**Pull request by tarang9211**

### CODE REVIEW

File: packages/get-packages/package.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "get-packages",
```
Comment (mitchellhamilton) [inline packages/get-packages/package.json]:
```suggestion
  "name": "@manypkg/get-packages",
```

File: packages/get-packages/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+export type Package = { packageJson: PackageJSON; dir: string };
+
+type Packages = {
+  tool: Tool;
+  packages: Package[];
+  root: Package;
+};
+
+export function getPackages(cwd: string): Promise<Packages>;
+export function getPackagesSync(cwd: string): Packages;
```
Comment (Noviny) [inline packages/get-packages/README.md]: This info seems not ideal as this is not a normal way of explaining imports.

This should really just be the import statements. Having the tool, package, and packages types is useful.

Speaking of, can we expose these as exports? Having access to types is nice.

Comment (tarang9211) [reply packages/get-packages/README.md]: Do you want me to add exports to the types?

File: packages/get-packages/src/index.test.ts [hunk 3]
```diff
@@ -0,0 +1,180 @@
+import fixturez from "fixturez";
+import { getPackages, getPackagesSync } from "./";
+
+const f = fixturez(__dirname);
+
+describe("getPackages", () => {
```
[the remark this answers is not in the document]
Comment (tarang9211) [reply packages/get-packages/src/index.test.ts]: I think this is done, right?

**Pull request merged**